relm4-examples/libadwaita/Cargo.toml

```diff
@@ -8,3 +8,5 @@ edition = "2021"
 rand = "0.8.4"
 relm4 = { path = "../../", features = ["libadwaita", "macros"] }
 relm4-components = { path = "../../relm4-components" }
+# For Sudoku example
+sudoku = "0.7.0"
```

Comment:
Maybe there is a better way than polluting the common toml. 
Maybe it's ok. what do you think?

Reply:
I think that's fine because sudoku seems to be a rather small dependency. But you can always add examples in their own crate (add a new folder and then `cargo init`) which also makes sense if you need multiple files.